tests/test_dse.py

```diff
@@ -655,7 +782,7 @@ def test_tti_rewrite_aggressive_opcounts(space_order, expected):
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('space_order,expected', [
-    (4, 183), (12, 375)
+    (4, 194), (12, 386)
```

Comment:
@mloubout here the increase is actually less than the 10% I had mentioned on slack